browser/ui/BUILD.gn

```diff
@@ -161,6 +161,13 @@ source_set("ui") {
       "views/toolbar/bookmark_button.h",
     ]
 
+    if (use_aura) {
```

Comment:
@AlexeyBarabash Can you check this condition again?
I think below views related sources should be included with `toolkit_views` condition above.
|use_aura| is false for macOS.